src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/dns/record_set/_create.py

```diff
@@ -50,7 +50,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--record-type"],
             help="The type of DNS record in this record set. Record sets of type SOA can be updated but not created (they are created when the DNS zone is created).",
             required=True,
-            enum={"A": "A", "AAAA": "AAAA", "CAA": "CAA", "CNAME": "CNAME", "MX": "MX", "NS": "NS", "PTR": "PTR", "SOA": "SOA", "SRV": "SRV", "TXT": "TXT"},
+            enum={"A": "A", "AAAA": "AAAA", "CAA": "CAA", "CNAME": "CNAME", "DS": "DS", "MX": "MX", "NAPTR": "NAPTR", "NS": "NS", "PTR": "PTR", "SOA": "SOA", "SRV": "SRV", "TLSA": "TLSA", "TXT": "TXT"},
```

Comment:
Q: Are we keeping NAPTR here?

Reply:
This is autogenerated code so it will contain NAPTR but we are not using it.